spra-play-server/src/test/scala/controllers/AdminControllerSpec.scala

```diff
@@ -532,6 +532,106 @@ class AdminControllerSpec extends PlayPostgresSpec with AdminUtils {
         .futureValue
       response.headOption.isEmpty must be(true)
     }
+
+    "don't fail when the column is citext or similar, but the value can be interpreted as Int." in withApiClient {
```

Comment:
When the value is CITEXT we should interpret it as string, any reason to interpret it as int?

Reply:
Before, if the value was "123", it was assumed that the value was an Int, and the same for the column type. This caused an internal server error because it compared a citext with an Int.

Reply:
From my point of view, the input argument must be handled like the type derived from the database instead of guessing its format.

Reply:
Yes, the changes in this pull request consider both the column type and the value to avoid this.